operator/option/config.go

```diff
@@ -201,6 +210,20 @@ const (
 	// Enabling this option reduces waste of IP addresses but may increase
 	// the number of API calls to AlibabaCloud ECS service.
 	AlibabaCloudReleaseExcessIPs = "alibaba-cloud-release-excess-ips"
+
+	// CiliumEndpointBatch options
+
+	// CEBUpsertSyncPeriod is the interval between reconciling newly created CEBs,
+	// modified CEBs or CEBs with new cilium endpoints with k8s-apiserver.
+	CEBUpsertSyncPeriod = "ceb-upsert-periodic-sync"
```

Comment:
Option should be called `ceb-upsert-sync-period` to be inline with the variable name.